`py -0p` fails

`py --list` works but `--list-paths` or `-0p` fail with a `TypeError`:

```ps1con
PS> py -0p
[ERROR] INTERNAL ERROR: TypeError: <lambda>() takes 1 positional argument but 2 were given
```

A

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -105,10 +105,19 @@ jobs:
       run: pymanager
 
     - name: 'Install default runtime'
-      run: pymanager install -vv default
+      run: pymanager install default
+      env:
+        PYMANAGER_DEBUG: true
 
     - name: 'List installed runtimes'
-      run: pymanager list -vv
+      run: pymanager list
+      env:
+        PYMANAGER_DEBUG: true
+
+    - name: 'List installed runtimes (legacy)'
+      run: pymanager --list-paths
+      env:
+        PYMANAGER_DEBUG: true
 
     - name: 'Launch default runtime'
       run: pymanager exec -m site
@@ -130,10 +139,12 @@ jobs:
 
     - name: 'Offline bundle download and install'
       run: |
-        pymanager list -vv --online 3 3-32 3-64 3-arm64
-        pymanager install -vv --download .\bundle 3 3-32 3-64 3-arm64
-        pymanager list -vv --source .\bundle
-        pymanager install -vv --source .\bundle 3 3-32 3-64 3-arm64
+        pymanager list --online 3 3-32 3-64 3-arm64
+        pymanager install --download .\bundle 3 3-32 3-64 3-arm64
+        pymanager list --source .\bundle
+        pymanager install --source .\bundle 3 3-32 3-64 3-arm64
+      env:
+        PYMANAGER_DEBUG: true
 
     - name: 'Remove MSIX'
       run: |

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -224,10 +224,20 @@ jobs:
     - powershell: |
         pymanager install -vv default
       displayName: 'Install default runtime'
+      env:
+        PYMANAGER_DEBUG: true
 
     - powershell: |
-        pymanager list -vv
+        pymanager list
       displayName: 'List installed runtimes'
+      env:
+        PYMANAGER_DEBUG: true
+
+    - powershell: |
+        pymanager --list-paths
+      displayName: 'List installed runtimes (legacy)'
+      env:
+        PYMANAGER_DEBUG: true
 
     - powershell: |
         pymanager exec -m site
@@ -249,11 +259,13 @@ jobs:
         PYMANAGER_DEBUG: true
 
     - powershell: |
-        pymanager list -vv --online 3 3-32 3-64 3-arm64
-        pymanager install -vv --download .\bundle 3 3-32 3-64 3-arm64
-        pymanager list -vv --source .\bundle
-        pymanager install -vv --source .\bundle 3 3-32 3-64 3-arm64
+        pymanager list --online 3 3-32 3-64 3-arm64
+        pymanager install --download .\bundle 3 3-32 3-64 3-arm64
+        pymanager list --source .\bundle
+        pymanager install --source .\bundle 3 3-32 3-64 3-arm64
       displayName: 'Offline bundle download and install'
+      env:
+        PYMANAGER_DEBUG: true
 
     - powershell: |
         Get-AppxPackage PythonSoftwareFoundation.PythonManager | Remove-AppxPackage

--- a/src/manage/list_command.py
+++ b/src/manage/list_command.py
@@ -207,7 +207,7 @@ FORMATTERS = {
     "prefix": format_bare_prefix,
     "url": format_bare_url,
     "legacy": format_legacy,
-    "legacy-paths": lambda i: format_legacy(i, paths=True),
+    "legacy-paths": lambda cmd, i: format_legacy(cmd, i, paths=True),
 }
 
 

